pkg/client/unversioned/transport.go

```diff
@@ -41,6 +41,7 @@ func (rt *userAgentRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 	}
 	req = cloneRequest(req)
 	req.Header.Set("User-Agent", rt.agent)
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
I don't think this is always correct... @smarterclayton any opinion on this re: binary data?